### PR TITLE
Style core elements

### PR DESCRIPTION
**Pull request by Vagyasri**
In this project, we added CSS styles to the Awesome books application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Awesome-Books
```
Comment (ZawadiSumba66) [inline README.md:1]: - [x]  Your README file currently lacks an MIT license. Please add it as recommended from this [template](https://github.com/microverseinc/readme-template)

File: style.css [hunk 2]
```diff
@@ -1,7 +1,122 @@
...
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.header h1 {
+  font-family: "Montserrat", sans-serif;
```
Comment (ZawadiSumba66) [inline style.css]: - [x] The `font-family: "Montserrat", sans-serif;` has been repeated severally in your code. Please ensure you assign an extra universal class in your HTML file where the style applies and use that one class in your css file. This makes your code DRY and easy to debug.
eg.
 ```
monterrat-font {
   font-family: "Montserrat", sans-serif;
}
```

- [x] Please ensure you apply these changes to the `font-weight: 200;`, `border: 1px solid orangered;` which have also been repeated.

File: style.css [hunk 3]
```diff
@@ -1,7 +1,122 @@
...
+  outline: none;
+  border: 1px solid orangered;
+  padding: 0.25em 0.5em;
+  color: red;
+  font-family: "Montserrat", sans-serif;
+  letter-spacing: 1.6px;
+  font-weight: 200;
+  font-size: 1.2rem;
+  transition: all 0.5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+  border-radius: 20%;
```
Comment (ZawadiSumba66) [inline style.css]: - [x] The add button should be positioned to the right and not the center as from the given layout.
- yours
image
- template
image

File: style.css [hunk 4]
```diff
@@ -1,7 +1,122 @@
...
+
+.removebtn:hover {
+  background: red;
+  border: 1px solid white;
+  color: white;
+  padding: 0.5em 1.5em;
+}
+
+form {
+  padding: 1em;
```
Comment (ZawadiSumba66) [inline style.css:84]: - [x] Please add a heading above your form as from the given design. You only have a heading on top of your books but it lacks on top of the form.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @Vagyasri && @elasfarc 👋 👋  , 

Good job so far on your project looks amazing! However, there are a few changes you need to make in order to move to the next step.

## Required Changes ♻️
_Check the comments under the review._ 👇 👇 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Vagyasri @elasfarc 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**